distribution/docs/src/main/resources/content/_running/available-console-commands.adoc

```diff
@@ -0,0 +1,44 @@
+:title: Available System Console Commands
+:type: maintaining
+:status: published
+:parent: Console Commands
+:summary: Types of console commands available.
+:order: 01
+
+== {title}
+
+Console commands follow a format of `namespace:command`.
+
+To get a list of commands, type in the namespace of the desired extension then press the *Tab* key.
+
+For example, type `catalog`, then press *Tab*.
+
+.[[available_console_commands]]Console Command Namespaces
+[cols="3,1,5" options="header"]
+|===
+|Title
+|Namespace
+|Description
+
+|${ddf-branding}:: Catalog :: Core :: Commands
+|<<catalog_command_descriptions, catalog>>
+|The Catalog Shell Commands are meant to be used with any `CatalogProvider` implementations.
+They provide general useful queries and functions against the Catalog API that can be used for debugging, printing, or scripting.
+
+|${ddf-branding} :: Admin :: Application Service
+|<<app_command_descriptions, app>>
+|The ${ddf-admin} Application Service contains operations to work with applications.
+
+|`${ddf-branding} :: Catalog :: Core :: PubSub Commands`
```

Comment:
The formatting of this cell doesn't match the others in the same column.

Reply:
fixed.